app/src/main/java/com/yeonae/chamelezone/data/source/remote/review/ReviewRemoteDataSourceImpl.kt

```diff
@@ -107,8 +109,61 @@ class ReviewRemoteDataSourceImpl(private val reviewApi: ReviewApi) : ReviewRemot
 
     }
 
-    override fun deleteReview() {
+    override fun deleteReview(placeNumber: Int, reviewNumber: Int, memberNumber: Int, callBack: ReviewCallBack<String>) {
+        val jsonObject = JsonObject().apply {
+            addProperty("memberNumber", memberNumber)
+        }
+        reviewService.deleteReview(placeNumber, reviewNumber, jsonObject).enqueue(object : Callback<ResponseBody>{
+            override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
+                Log.d("tag", t.toString())
```

Comment:
```suggestion
        reviewService.deleteReview(placeNumber, reviewNumber, jsonObject)
             .enqueue(object : Callback<ResponseBody>{
                 override fun onFailure(call: Call<ResponseBody>, t: Throwable) {
                Log.d("tag", t.toString())
```
chaning convetion을 준수해주세요

Reply:
수정했습니다
ea03568